WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift

```diff
@@ -288,7 +288,7 @@ private class AccountSettingsController: SettingsController {
             return NSLocalizedString("You're not authorized to close the account.",
                                      comment: "Error message displayed when unable to close user account due to being unauthorized.")
         case "atomic-site":
-            return NSLocalizedString("This user account cannot be closed while it has active atomic sites.",
+            return NSLocalizedString("This user account could not be closed. Please contact us by going to Me > Help & Support screen.",
```

Comment:
Personal note: I'd have said "can't be closed automatically" or something like this, to avoid making the user think that "it won't be possible to close it at all" but instead tell them "you need to contact us so we can do some stuff in our end to close it", which would be less worrying than "can't be closed [at all]" imho 🤷‍♂️